### PR TITLE
Getting Started with Calendar: Add a tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/00 Getting Started with Calendar.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include tutorials-intro-installationnote
+
+The Calendar is a component that allows users to select the required date within a specified date range.
+
+This tutorial shows how to add a Calendar component to your application and configure its core features.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/00 Getting Started with Calendar.md]: If one-sentence paragraphs can be avoided, it is better to do it.

```suggestion
The Calendar component allows users to select a required date within a specified date range. This tutorial shows how to add this component to your application and configure its core features.
```

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+To specify which calendar view (month, year, decade or century) should be displayed at first, set the [zoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#zoomLevel) property. To make certain calendar views inaccessible, specify the [maxZoomLevel](\/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#maxZoomLevel) and [minZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#minZoomLevel) properties.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md]:
```suggestion
To specify which calendar view (month, year, decade, or century) should be displayed initially, set the [zoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#zoomLevel) property. To make certain calendar views inaccessible, specify the [maxZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#maxZoomLevel) and [minZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#minZoomLevel) properties.
```

Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md]: maxZoomLevel and minZoomLevel are not demonstrated in this tutorial. This should be stated in the description.

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        </DxCalendar>
+    </template>
+
+    <script>
+    // ...
+    </script>
+
+##### React
+
+When using React, it is necessary to add the `default` prefix to the **zoomLevel** property to control the change of the component state. For more information, refer to the following article: [Uncontrolled Mode](/Documentation/Guide/React_Components/State_Management/#Uncontrolled_Mode).
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md]: I suggest moving this note after the code.

```suggestion
In the code above, the **zoomLevel** property has the `default` prefix. This prefix specifies that we will not handle property changes, and the Calendar should handle them internally. For more information, refer to the following article: [Uncontrolled Mode](/Documentation/Guide/React_Components/State_Management/#Uncontrolled_Mode).
```

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify the Value Range.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#max) properties to specify the range of available dates.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify the Value Range.md]: How about we add informtation about [disabledDates](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) and rename the article? "Specify Available Dates"?

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+            new Date('2022-11-11T00:00:00.000Z'),
+            new Date('2022-11-24T00:00:00.000Z'),
+            new Date('2022-12-25T00:00:00.000Z')
+        ];
+        
+        getCellCssClass(date: any, view: any) {
+            let cssClass = '';
+            console.log(date, view);
+            this.federalHolidays.forEach((item) => {
+            if (date.getDate() === item.getDate() && date.getMonth() === item.getMonth() && view !== 'year') {
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md]: Indentation

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+            new Date('2022-09-05T00:00:00.000Z'),
+            new Date('2022-10-10T00:00:00.000Z'),
+            new Date('2022-11-11T00:00:00.000Z'),
+            new Date('2022-11-24T00:00:00.000Z'),
+            new Date('2022-12-25T00:00:00.000Z')
+        ];
+        
+        getCellCssClass(date: any, view: any) {
+            let cssClass = '';
+            console.log(date, view);
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md]: console.log

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+                // ...
+                cellTemplate: 'custom'
+            }
+        },
+        methods: {
+            getCellCssClass(date, view) {
+                let cssClass = '';
+
+                federalHolidays.forEach((item) => {
+                if (date.getDate() === item.getDate() && date.getMonth() === item.getMonth() && view !== 'year') {
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md]: Indentation

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Add a Today Button.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+Set the [showTodayButton](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#showTodayButton) property to **true** to add a **Today** button that selects the current date.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Add a Today Button.md]: To remove the repetition of "to":

```suggestion
Enable the [showTodayButton](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#showTodayButton) property to add a **Today** button that selects the current date.
```

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md (hunk 5 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md]: Noted

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md (hunk 6 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md]: Noted

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md (hunk 7 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Calendar/00 Getting Started with Calendar/20 Customize Cell Appearance.md]: Noted

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md (hunk 2 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Zoom Level.md]: Noted

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify the Value Range.md (hunk 4 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify the Value Range.md]: Noted

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Available Dates.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
+Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#max) properties to specify the range of available dates. You can also use the [disabledDates](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) property to disable certain dates. For example, the code below disables all the Sundays in the Calendar and specifies the range of dates to one year.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Available Dates.md]:
```suggestion
Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#max) properties to specify the range of available dates. You can also use the [disabledDates](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) property to disable certain dates. For example, the code below disables all Sundays and limits the range of dates to year 2022.
```

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify Available Views.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
+To specify which calendar view (month, year, decade, or century) should be displayed initially, set the [zoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#zoomLevel) property. To make certain calendar views inaccessible, specify the [maxZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#maxZoomLevel) and [minZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#minZoomLevel) properties. The **maxZoomLevel** and **minZoomLevel** properties are not demonstrated in this tutorial.
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/15 Specify Available Views.md]: To get rid of repetition:

```suggestion
To specify which calendar view (month, year, decade, or century) should be displayed initially, set the [zoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#zoomLevel) property. To make certain calendar views inaccessible, specify the [maxZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#maxZoomLevel) and [minZoomLevel](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#minZoomLevel) properties (not demonstrated in this tutorial).
```

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Customize Cell Appearance.md [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+        new Date('2022-11-24T00:00:00.000Z'),
+        new Date('2022-12-25T00:00:00.000Z')
+    ];
+
+    export default {
+        // ...
+        data() {
+            return {
+                // ...
+                cellTemplate: 'custom'
```
Comment (RomanTsukanov) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Customize Cell Appearance.md]: You shouldn't need this, `cell-template="custom"` should work.

Comment (vladaskorohodova) [reply concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Customize Cell Appearance.md]: Noted

**Review by RomanTsukanov — APPROVED** [review]

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/00 Getting Started with Calendar.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
+#include tutorials-intro-installationnote
+
+The Calendar component allows users to select a required date within a specified date range. This tutorial shows how to add this component to your application and configure its core features.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/00 Getting Started with Calendar.md]:
```suggestion
The Calendar component allows users to select a date within a specified date range. This tutorial shows how to add this component to your application and configure its core features.
```

I'm not sure why you add `required`

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Available Dates.md [hunk 13]
```diff
@@ -0,0 +1,91 @@
+Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#max) properties to specify the range of available dates. You can also use the [disabledDates](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) property to disable certain dates. For example, the code below disables all Sundays and limits the range of dates to year 2022.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/10 Specify Available Dates.md]:
```suggestion
Use the [min](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#max) properties to specify the range of available dates. You can also use the [disabledDates](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#disabledDates) property to disable certain dates. For example, the code below disables all Sundays and limits the range of dates to 2022.
```

File: concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Customize Cell Appearance.md [hunk 14]
```diff
@@ -0,0 +1,210 @@
+Use [cellTemplate](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#cellTemplate) to customize cell appearance. For example, in the following code all the dates that are federal US hoildays become red in the `month` view.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Calendar/00 Getting Started with Calendar/25 Customize Cell Appearance.md]:
```suggestion
Use [cellTemplate](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#cellTemplate) to customize cell appearance. For example, in the following code all the dates that are federal US hoildays become red in the `month` view:
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**